kubernetes/README.md

```diff
@@ -107,15 +111,12 @@ curl --header "Authorization: Bearer TOKEN_GOES_HERE" https://<domain>/scim/User
 
 You can now continue with the administration guide to configure your Identity Provider to enable provisioning with your SCIM bridge.
 
-## Upgrading
+## Updating
 
-To upgrade your SCIM bridge, follow these steps:
+To update SCIM bridge, connect to your Kubernetes cluster and run the follwing command:
```

Comment:
```suggestion
To update the SCIM bridge, connect to your Kubernetes cluster and run the following command:
```

Reply:
This is on purpose (dropping the definite article) and in line with [our style guide](https://support.1password.com/style-guide/#1password-scim-bridge) for referring to product names. 😃

Reply:
Good callout. However `following` is misspelt.